syntax = "proto3";
package orders;

message OrderCreated {
  string order_id = 1;
  string user_id = 2;
  double amount = 3;
}

message ReserveInventory {
  string order_id = 1;
}
